pkgs/development/compilers/julia/precompile_succeeded.patch

```diff
@@ -0,0 +1,156 @@
+{ fetchzip, fetchFromGitHub, lib, pkgs, stdenv
```

Comment:
This file looks the same as 1.6.nix. Am I missing something?

Reply:
I am currently traveling so I didn't have time to compare this with the other pull request. I was thinking maybe bits and bobs of this one might be still useful to someone.

Reply:
No rush! I just meant that this doesn't look like a patch file. It looks more like the 1.6.nix file from this PR, but perhaps I'm missing something!